.github/workflows/workflow.yaml

```diff
@@ -114,7 +114,7 @@ jobs:
           make require-suites
   require_sync:
     runs-on: ubuntu-latest
-    name: "Require a sync.yaml file and metadata.gatekeeper.sh/requiresSyncData annotation for every template.yaml using data.inventory"
+    name: "Require a sync.yaml file and metadata.gatekeeper.sh/requires-sync-data annotation for every template.yaml using data.inventory"
```

Comment:
How does the new annotation work with older GK versions since `metadata.gatekeeper.sh/requires-sync-data` is only introduced after https://github.com/open-policy-agent/gatekeeper/releases/tag/v3.13.0-beta.1? and how does the old annotation work with newer GK versions?

Reply:
I see CI is not broken even tho the GK versions we are testing with do not know about the new annotation.

Reply:
Hi @ritazh~ - The `requiresSyncData` annotation was introduced to gatekeeper-library with a "synthetic" require-suites test, but the annotation is not (yet~) used by GK/CI Tests.  This change is to have the gatekeeper-library repo ready in advance of GK 3.13.

I have https://github.com/open-policy-agent/gatekeeper-library/issues/345 open to use/test the future GK 3.13 `requiresSyncData` annotation unmarshall.